mindsdb/api/mysql/mysql_proxy/executor/executor_commands.py

```diff
@@ -1256,6 +1255,18 @@ def answer_drop_view(self, statement):
 
         return ExecuteAnswer(answer_type=ANSWER_TYPE.OK)
 
+    def _create_persistent_chroma(self, project_name, kb_name, engine="chromadb"):
+        """Create default vector database for knowledge base, if not specified"""
+
+        vector_store_name = f"{kb_name}_{engine}"
+
+        persist_directory = f"{self.session.config.paths['storage']}/vector_databases//{project_name}/{vector_store_name}"
```

Comment:
Should be fine for local but don't imagine its going to play well on cloud!

Whats the best parameter to check to see if its cloud or local run?

How is user data currently partitioned on cloud?